vhd/lib/vhd-util-scan.c

```diff
@@ -1255,13 +1255,17 @@ vhd_util_scan_find_volume_targets(int cnt, char **names,
 	targets   = NULL;
 
 	err = lvm_scan_vg(volume, &vg);
-	if (err)
+	if (err) {
+		fprintf(stderr, "lvm_scan_vg failed %s\n", err);
```

Comment:
shouldn't this be %d?

Reply:
Yes and it failed to compile, I obviously updated this after the last compile. Fixed now.

Reply:
might be useful to add some github actions to try and build blktap, but getting all the dependencies would be tricky, they probably won't be neatly available in distro packages.

Reply:
The dependencies won't be available at all in the distros, as they require the frankentap kernel device and headers which only exist in the Citirx Hypervisor kernel patchqueue.